specifyweb/frontend/js_src/lib/components/WbPlanView/helpers.ts

```diff
@@ -81,7 +83,7 @@ export async function savePlan({
       expectedResponseCodes: [Http.NO_CONTENT],
     }
   ).then(async () =>
-    newlyAddedHeaders.length === 0
+    newlyAddedHeaders.length === 0 && !isModified
```

Comment:
could you rename `isModified` to `columnCountChanged` to be more descriptive?
`isModified` is a wrong name as mapping could have been modified too - but the variable is only true if column count changed